migrations/etl/1_Mine_profile_detail_location.sql

```diff
@@ -108,7 +108,7 @@ BEGIN
         create_timestamp    ,
         update_user         ,
         update_timestamp    ,
-        major               )
+        major_mine_ind               )
```

Comment:
Thanks for fixing ETL for me! Should probably line up the closing parenthesis with the commas, though